library/modules/bigip_device_info.py

```diff
@@ -12534,7 +12538,15 @@ def address(self):
     @property
     def netmask(self):
         parts = self._values['address'].split('/')
-        return to_netmask(parts[1])
+        if is_valid_ip(parts[0]):
+            addr = ip_address(u'{0}'.format(parts[0]))
+            if addr.version == 4:
+                ip = ip_interface(str(self._values['address']))
+                result = ip.netmask
```

Comment:
do not use str do this:
ip_interface('u{0}'.format(self._values['address']))

str in py3 != str in py2

Reply:
sure!